test/cmdlineTests/asm_json_import_missing_subobjects_indices/stdin

```diff
@@ -0,0 +1,22 @@
+{
+  ".code": [
+    {
+      "name": "PUSH",
+      "value": "0"
+    }
+  ],
+  ".data": {
+    "2": {
+      ".code": []
+    },
+    "0": {
+      ".code": []
+    },
+    "4": {
+      ".code": []
+    },
+    "1": {
+      ".code": []
+    }
```

Comment:
Please add a few more objects to go over 10. I suspect that the current code may actually be broken because it uses `stoi` for integer parsing, which by default uses base-10 and not base-16. I thought it must be fine because `objectCompiler/manySubObjects.yul` has many objects and works, but that may be just because it's not going through the import.

Reply:
I added more objects and it seems to be working correctly now.

Reply:
I still think this can't be right. We clearly use base-16 when we create the keys: https://github.com/ethereum/solidity/blob/1b5775acff81e1a049c809ac2503afae75a546d0/libevmasm/Assembly.cpp#L289-L290

The test passes because both it and the import code use base-10. But I think that if you exported assembly with more than 10 subassemblies, you'd get something that would then fail to import because of the hex numbers.

I wonder if I can actually test that. I'm not sure if it's even possible to have Solidity code that has more than 2 assemblies and for Yul we don't have the `--asm-json` option merged yet.

Reply:
Looks like it can be tested from Solidity after all. `new C()` adds new subassemblies (I expected them to be embedded as data blobs instead).

And yeah, just like I expected, this fails export+import:
```solidity
contract C0 {}
contract C1 {}
contract C2 {}
contract C3 {}
contract C4 {}
contract C5 {}
contract C6 {}
contract C7 {}
contract C8 {}
contract C9 {}
contract C10 {}

contract D {
    function run() public {
        new C0();
        new C1();
        new C2();
        new C3();
        new C4();
        new C5();
        new C6();
        new C7();
        new C8();
        new C9();
        new C10();
    }
}
```
```
solc test.sol --asm-json --json-indent 4 > asm.json
# Manually remove all but D from asm.json
solc --import-asm-json asm.json
```
```
Error: Assembly Import Error: The key 'a' inside '.data' is not an integer.
```